picard/collection.py

```diff
@@ -106,23 +106,24 @@ def request_finished(document, reply, error):
                 echo=log.error
             )
             return
-        collection_list = document.metadata[0].collection_list[0]
-        if "collection" in collection_list.children:
+        if document and "collection" in document:
```

Comment:
I remember getting an error during one of my testing runs on this. That's why I made the change. Probably in case of an error or api limit reached or when the reply was not parsed correctly.